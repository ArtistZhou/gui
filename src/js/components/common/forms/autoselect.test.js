import React from 'react';
import renderer from 'react-test-renderer';
import AutoSelect from './autoselect';

it('renders correctly', () => {
  const tree = renderer.create(<AutoSelect items={[]} label="test" />).toJSON();
  expect(tree).toMatchSnapshot();
});
