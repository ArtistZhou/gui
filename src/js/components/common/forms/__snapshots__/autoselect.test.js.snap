// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiFormControl-root-1"
>
  <label
    className="MuiFormLabel-root-16 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13"
    data-shrink={false}
    htmlFor="adornment-target-select"
  >
    test
  </label>
  <div
    className="MuiInputBase-root-36 MuiInput-root-23 MuiInput-underline-27 MuiInputBase-formControl-37 MuiInput-formControl-24"
    onClick={[Function]}
    style={
      Object {
        "minWidth": "250px",
      }
    }
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input-46 MuiInput-input-31"
      disabled={false}
      id="adornment-target-select"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      placeholder="test"
      required={false}
      type="text"
      value=""
    />
  </div>
</div>
`;
